### DISCUSSION TITLE
IIP-12: Locking invitee rewards

### DISCUSSION THREAD

**Discussion by midenaio** [idena-network/idena-docs#149] [IIPs]
Lock invitee rewards for 10 subsequent epochs

Read more details [here](https://docs.idena.io/docs/iip/iip-12).

**Comment by teavar** [comment]
the link to more details does not work,
should be https://idena-docs-git-iip-12-idena.vercel.app/docs/iip/iip-12

**Reply by midenaio** [reply]
IIP-12 has been merged. The link is working now.

**Comment by KPHEMRAJ** [comment]
Why only invitee rewards to be locked? The inviter s reward should also be locked cz there are cases where inviter with big stake self invites.

**Reply by midenaio** [reply]
Thanks for your feedback. 

An inviter with a large stake has no motivation to terminate their account and start all over again just to sell an invitation reward. Otherwise after termination the inviter would need to wait for another 10 epochs to reach 100% stake protection and stake their large amount safely again.

**Reply by KPHEMRAJ** [reply]
Sorry if you mis understood the question. For ex : i have an identity with huge stake, suppose i self invite for my 2nd account, currently i get invite rewards for both invitee and inviter , if i pass 3 validations successfully. After iip 12 , if i wish to kill my second account, I still get my inviter rewards after 3 epochs. There still would exist some misuse. So my proposal is to keep that invite reward locked for both

**Reply by teavar** [reply]
@KPHEMRAJ , for how many epochs would you suggest to lock the inviters part of the rewards?

**Reply by midenaio** [reply]
@KPHEMRAJ Got you. However IIP-12 is only aimed to discourage abusers who hunt for invitations from identities with large stakes.
We acknowledge that self-invitation is a difficult issue, but a new proposal should be created to discuss it.
Could you open a new discussion for that?

**Reply by KPHEMRAJ** [reply]
@teavar @midenaio maybe we will pause this discussion for now and  check if too many people misuse self invite after iip 12. Because currently it might be less , but when very less people want invite, self invite may increase. In that case we will have further discussion

**Comment by 0JustinMiles0** [comment]
Will invitee rewards paid before the fork activation be subjected to the 10 epochs lock in period or only those paid after the fork activation?

**Reply by Zen-44** [reply]
Only ones after the fork since the current protocol is still used.
